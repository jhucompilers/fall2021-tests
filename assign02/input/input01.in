var x;
x = readint();
println(x);
x;
