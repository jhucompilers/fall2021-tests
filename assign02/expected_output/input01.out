42
Result: 42
